### PR TITLE
fix footer to bottom of screen

### LINKED ISSUE #2115: Footer not fixed to bottom of screen
The footer isn't fixed to the bottom of the screen. See example in the screenshot below:

image

This appears to be the case on all pages where the footer appears including the following pages:

https://app.couchers.org/ (dashboard)
https://app.couchers.org/contribute
https://app.couchers.org/donate

### PR DESCRIPTION
**Pull request by wginsberg**
closes #2115 

**Web frontend checklist**
- [x] Formatted my code with `yarn format && yarn lint --fix`
- [x] There are no warnings from `yarn lint`
- [x] There are no console warnings when running the app
- [x] Added any new components to storybook
- [x] Added tests where relevant
- [x] All tests pass
- [x] Clicked around my changes running locally and it works
- [x] Checked Desktop, Mobile and Tablet screen sizes

Screen Shot 2022-04-29 at 1 54 41 PM

### CODE REVIEW

File: app/web/components/AppRoute.tsx [hunk 1]
```diff
@@ -48,6 +50,14 @@ export const useAppRouteStyles = makeStyles((theme) => ({
       )})`,
     },
   },
+  "@global body": {
+    height: "100%",
```
Comment (darrenvong) [inline app/web/components/AppRoute.tsx:54]: I think we had this a while back (setting 100% height on html and body) but do you happen to remember why we removed it @lucaslcode?

The only thing I remember is the weird background problem where the background colour of body ends at 100% height even when the inner stuff is taller than that, but this solution doesn't appear to suffer from that so maybe this has always been the way to go 🤔

Comment (lucaslcode) [reply app/web/components/AppRoute.tsx:54]: Can't remember!

**Review by lucaslcode — CHANGES_REQUESTED** [review]
On mobile, I get some kind of padding on the right (you can see backgrounds don't cover the whole horizontal width).

It seems to be caused by the card style in `LandingPage.tsx`:
```
  card: {
    margin: theme.spacing(0, -4),
    padding: theme.spacing(4),
  },
```

I remember this was a trick to get the card to be full-width on mobile. I guess giving html and body height makes this not work anymore. Happy with any other solution that looks okay!

**Review by lucaslcode — APPROVED** [review]
Apologies, looks like that didn't originate in this PR

**Pull request merged**